src/PageTypes/BaseHomePage.php

```diff
@@ -50,7 +83,7 @@ public function getCMSFields() {
 				TreeDropdownField::create(
 					'LearnMorePageID',
 					_t('BaseHomePage.LearnMoreLink','Page to link the "Learn More" button to:'),
```

Comment:
Translations still unaltered

Reply:
I might do this a bit further down the line if that's OK?